includes/admin/settings/register-settings.php

```diff
@@ -2183,18 +2188,24 @@ function edd_sendwp_callback($args) {
 	// Output the appropriate button and label based on connection status
 	if( $client_connected ) :
 		?>
+		<div class="inline notice notice-success">
+			<p><?php _e( 'SendWP plugin activated.', 'easy-digital-downloads' ); ?> <?php echo $forwarding_enabled ? $connected : $disconnected ; ?></p>
 
-		<p>
-			<button id="edd-sendwp-disconnect" class="button"><?php _e( 'Disconnect SendWP', 'easy-digital-downloads' ); ?></button>
-		</p>
-		<p>Your site is connected to SendWP. <?php echo $forwarding_enabled ? $connected : $disconnected ; ?></p>
-
+			<p>
+				<button id="edd-sendwp-disconnect" class="button"><?php _e( 'Disconnect SendWP', 'easy-digital-downloads' ); ?></button>
+			</p>
+		</div>
 		<?php
 	else :
 		?>
-
 		<p>
-			<button type="button" id="edd-sendwp-connect" class="button button-primary"><?php esc_html_e( 'Connect SendWP', 'easy-digital-downloads' ); ?>
+			<?php _e( 'We recommend SendWP to ensure quick and reliable delivery of all emails sent from your store, such as purchase receipts, subscription renewal reminders, password resets, and more.', 'easy-digital-downloads' ); ?>
+		</p>
+		<p>
+			<?php printf( __( '%sLearn More%s (Free trial available)', 'easy-digital-downloads' ), '<a href="https://sendwp.com/" taregt="_blank" rel="noopener noreferrer">', '</a>' ); ?>
```

Comment:
```suggestion
			<?php printf( __( '%sLearn More%s (Free trial available)', 'easy-digital-downloads' ), '<a href="https://sendwp.com/" target="_blank" rel="noopener noreferrer">', '</a>' ); ?>
```

Reply:
Also, I don't see a free trial available? According to their website you can:

> Try it for 14 days for only $1.00

So it's more like a paid trial.

Reply:
![image](https://user-images.githubusercontent.com/1390124/112371294-afab7100-8c9b-11eb-9c53-80e4b54281a5.png)

![image](https://user-images.githubusercontent.com/1390124/112371320-b89c4280-8c9b-11eb-8598-b53c3b773dce.png)

Reply:
To be clear, Recapture has a free trial, but SendWP does not. I was incorrect earlier.